netgrasp/netgrasp.py

```diff
@@ -437,7 +437,8 @@ def log_event(mid, iid, did, rid, event, have_lock = False):
         ng.debugger.debug("entering log_event(%s, %s, %s, %s, %s, %s)", (mid, iid, did, rid, event, have_lock))
 
         # Only log events for which there are subscribers.
-        if ((ng.email["enabled"] and event in ng.email["alerts"]) or (ng.notification["enabled"] and event in ng.notification["alerts"])):
+        if (ng.email["enabled"] and event in ng.email["alerts"]) \
+            or (ng.notification["enabled"] and event in ng.notification["alerts"]):
```

Comment:
> That's a great point and why I thought a bit before making the change. I want you to know that the last thing I want to do is "pep8 you", i.e. send you a bunch pep8 changes. :) That's not my intention, although it may appear that way or actually be the case for some changes I submit (if I run across them while fixing something else). I'd like to try and explain my motivation for that.

I'm a big fan of consistent code-style. Choosing the same coding-style as the Python standard library makes sense to me. I'm okay w/ embracing pep8 for this project.

My day-job involves a lot of cleaning up other people's code; consistent coding-style is critical. Any inconsistencies you see in my code are because I'm teaching myself Python as I go, and am carrying with me coding-styles from other languages that aren't necessarily sensical in Python. Cleanups remain very welcome, that is why I'm quickly reviewing and merging your PR's.

> My motivation is to use the code on my home network and have a tool for network visibility and to learn/contribute to the project when I have time. So with regards to the contribute part, my overall motivation is code readability and maintenance. I'll try to explain more below.

How is netgrasp working on your local network? I assume you have it up and running, and are receiving email alerts and digests correctly? I've been using netgrasp to monitor my home network for a month or so now and am generally happy with it (though there's always room for improvement). It's very quick to notify me of new devices joining the network which has proven helpful in multiple ways.

Don't hold back on opening issues for features you'd like to see added / plan to add yourself.

> One of the things I've read and learned is code is read a lot more than it's written. Especially when you're coming into a new code base/project. So I've been reading a lot of the code and trying to process how it all fits together. When I'm doing that and come across a long line (>120 characters), I need to scroll horizontally to read it and it takes me out of context a little bit. I can't see all of the code on the page.

While per my earlier comment I disagree long lines are a problem, I see that Pep8 says "Limit all lines to a maximum of 79 characters." It goes on to add, "For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters)". As I noted before, I'm on-board with embracing the pep8 coding standards; I'm not crazy about force-wrapping lines, but if we do it seems we should be consistent with the standards and do so at 99 characters.

It's going to be a fair amount of code to consistently and correctly wrap; I'd prefer to consider this a very low-priority cleanup.

> Sorry for the long comment, but I wanted to try and explain. In a nutshell, for me it's about readability and being able to understand the code and spot bugs, which also folds into maintainability.

Not at all, I appreciate the feedback and understanding your motivations. 

For me, this is a project with which I'm learning Python; as well as something I generally find useful. I enjoy collaboration and welcome your patches/comments/etc!